rails-commenting.rb

```diff
@@ -33,12 +38,14 @@ def create
 
   def edit
     # ---6)
+    # edits the blog post based on the matching id
     @post = BlogPost.find(params[:id])
   end
 
   def update
     @post = BlogPost.find(params[:id])
     # ---7)
+    # Updates blog post entry if the id exist in the array. which would mean .valid? return a true boolean value
```

Comment:
Not if the id exists in the array, but if the updated post is valid according to the validations set and the strong params.